inference-engine/tests/functional/inference_engine/lp_transformations/is_asymmetric_on_weights.cpp

```diff
@@ -0,0 +1,149 @@
+    element::Type,
+    ngraph::PartialShape,
+    IsAsymmetricOnWeightsTestValues,
+    std::pair<std::vector<bool>, bool> > IsAsymmetricOnWeightsParams;
+
+class IsAsymmetricOnWeightsTransformation : public LayerTransformation, public testing::WithParamInterface<IsAsymmetricOnWeightsParams> {
+public:
+    void SetUp() override {
+        const auto netPrecision = std::get<0>(GetParam());
+        const auto inputShape = std::get<1>(GetParam());
+        auto testValues = std::get<2>(GetParam());
+        std::pair<std::vector<bool>, bool> transposeAndIsAsymmetricOnWeights = std::get<3>(GetParam());
+
+        actualFunction = ngraph::builder::subgraph::ConvolutionFunction::getOriginal(
+            netPrecision,
+            testValues.actual.precisionBeforeDequantization,
+            inputShape,
+            testValues.actual.dequantizationOnActivations,
+            testValues.actual.weights,
+            testValues.actual.fakeQuantizeOnWeights,
+            transposeAndIsAsymmetricOnWeights.first[0],
+            transposeAndIsAsymmetricOnWeights.first[1],
+            transposeAndIsAsymmetricOnWeights.first[2],
+            transposeAndIsAsymmetricOnWeights.first[3],
+            transposeAndIsAsymmetricOnWeights.first[4]);
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<IsAsymmetricOnWeightsParams> obj) {
+        const auto netPrecision = std::get<0>(obj.param);
+        auto inputShape = std::get<1>(obj.param);
+        IsAsymmetricOnWeightsTestValues testValues = std::get<2>(obj.param);
+        std::pair<std::vector<bool>, bool> transposeAndIsAsymmetricOnWeights = std::get<3>(obj.param);
+
+        std::ostringstream result;
+        result << toString(testValues.params) << "_" <<
+            netPrecision << "_" <<
+            inputShape << "_" <<
+            testValues.actual.precisionBeforeDequantization << "_" <<
+            testValues.actual.dequantizationOnActivations << "_" << "_weights_" <<
+            testValues.actual.weights->get_element_type() << "_" << "{ " <<
+            testValues.actual.weights->cast_vector<float>()[0] << " }_" <<
+            testValues.actual.fakeQuantizeOnWeights << "_" <<
+            transposeAndIsAsymmetricOnWeights.first[0] << "_" <<
+            transposeAndIsAsymmetricOnWeights.first[1] << "_" <<
+            transposeAndIsAsymmetricOnWeights.first[2] << "_" <<
+            transposeAndIsAsymmetricOnWeights.first[3] << "_" <<
+            transposeAndIsAsymmetricOnWeights.first[4];
+        return result.str();
+    }
+};
+
+TEST_P(IsAsymmetricOnWeightsTransformation, CompareFunctions) {
+    actualFunction->validate_nodes_and_infer_types();
+
+    std::shared_ptr<ov::Node> convolution;
+    const auto ops = actualFunction->get_ops();
+    for (const auto& op : ops) {
+        if (ov::is_type<ov::op::v1::Convolution>(op)) {
+            convolution = op;
+            break;
+        }
+    }
+    ASSERT_TRUE(convolution != nullptr) << "convolution was not found";
```

Comment:
Looks like we can use existing method:
```suggestion
    const auto conv_vec = LayerTransformation::get<opset1::Convolution>(actualFunction);
    ASSERT_TRUE(!conv_vec.empty()) << "convolution was not found";
    const auto convolution = conv_vec[0];
```

Reply:
done, thanks